grafana-plugin/src/containers/GSelect/GSelect.tsx

```diff
@@ -61,7 +61,6 @@ const GSelect = observer((props: GSelectProps) => {
     showWarningIfEmptyValue = false,
     getDescription,
     filterOptions,
-    // fromOrganization,
```

Comment:
this is commented out, ie. never used, removed the few leftover references to it.